sidebars.js

```diff
@@ -166,9 +166,9 @@ module.exports = {
   },
   {
     "References": [
-      "reference/cli",
-      "reference/manifest",
-      "reference/compose",
+      "reference/okteto-cli",
+      "reference/okteto-manifest",
+      "reference/docker-compose",
       "reference/helm-chart-settings",
```

Comment:
I would go with `helm-chart-values` here, `values.yml` is the standard name of this file